### ISSUE TITLE
`include-only-table` filter does not allow long table names

### ISSUE THREAD

**Issue by KubaSzostak** [dimitri/pgcopydb#413]
`include-only-table` filter does not allow entries longer than 64 bytes. Length validation occurs on combined schema name and table instead of schema name and table name separately. 

**Steps to reproduce**

Create database content using long table name

```sql
create schema schema_name_20_chars;

create table schema_name_20_chars.very______long______table______name_______50_chars (id serial);
```

Create filter.ini file

```ini
[include-only-table]
schema_name_20_chars.very______long______table______name_______50_chars
```

Execute `clone` command

```console
pgcopydb clone --no-owner --no-acl --no-comments --restart --skip-extensions --filter filter.ini
```

Output: ` pgcopydb and Postgres only support names up to 64 bytes`.

```console
20:00:43 11351 INFO   Running pgcopydb version 0.13-1.pgdg20.04+1 from "/usr/bin/pgcopydb"
20:00:43 11351 INFO   [SOURCE] Copying database from "postgres://pg_admin%4migration_test.postgres.database.azure.com:5432/migration_test"
20:00:43 11351 INFO   [TARGET] Copying database into "postgres://pg_admin@flexible-db-migration-test.postgres.database.azure.com:5432/migration_test"
20:00:43 11351 ERROR  Failed to parse schema name "schema_name_20_chars" (21 bytes long), pgcopydb and Postgres only support names up to 64 bytes
20:00:43 11351 ERROR  Failed to parse filters in file "filter.ini"
```

---

For comparison execute the same migration without `--filter filter.ini` parameter

```
pgcopydb clone --no-owner --no-acl --no-comments --restart --skip-extensions 
```

Output: Success

```
      OID |               Schema |                                               Name | copy duration | indexes | create index duration
----------+----------------------+----------------------------------------------------+---------------+---------+----------------------
168342579 | schema_name_20_chars | very______long______table______name_______50_chars |          30ms |       0 |                   0ms


                                               Step   Connection    Duration   Concurrency
 --------------------------------------------------   ----------  ----------  ------------
                                        Dump Schema       source        82ms             1
   Catalog Queries (table ordering, filtering, etc)       source       915ms             1
                                     Prepare Schema       target        95ms             1
      COPY, INDEX, CONSTRAINTS, VACUUM (wall clock)         both       516ms         1 + 5
                                  COPY (cumulative)         both        30ms             1
                         Large Objects (cumulative)         both       335ms             1
             CREATE INDEX, CONSTRAINTS (cumulative)       target         0ms             4
                                    Finalize Schema       target        75ms             1
 --------------------------------------------------   ----------  ----------  ------------
                          Total Wall Clock Duration         both       1s964         1 + 5
 --------------------------------------------------   ----------  ----------  ------------
```

**Comment by dimitri** [comment]
Hi @KubaSzostak ; thanks for reporting the issue. I believe it should be fixed now, please consider building pgcopydb from fresh sources and try again.

**Comment by dimitri** [comment]
It looks like somehow you're still using the previous binary for pgcopydb, not the newly built one?

**Comment by KubaSzostak** [comment]
That's my pgcopydb version. Should it be newer?
```console
pgcopydb --version
14:41:15 22892 INFO   Running pgcopydb version 0.13-1.pgdg20.04+1 from "/usr/bin/pgcopydb"
pgcopydb version 0.13-1.pgdg20.04+1
compiled with PostgreSQL 15.3 (Ubuntu 15.3-1.pgdg20.04+1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0, 64-bit
compatible with Postgres 10, 11, 12, 13, 14, and 15
```

**Comment by dimitri** [comment]
Yes, it seems you're using a version that was released last week, which does not include today's commits. If you don't build from source or use the GitHub CI docker image, then you won't have today's commits in your binary. See also https://pgcopydb.readthedocs.io/en/latest/install.html#docker-images and https://pgcopydb.readthedocs.io/en/latest/install.html#build-from-sources.

**Comment by KubaSzostak** [comment]
I wasn't able to build `pgcopydb` from the sources on Ubuntu because of `cannot find -lzstd` error. I ended with docker image. I confirm that both issues (#412 and #413) are fixed in version 0.13.5.g452ce59. Thank you!
